ci/scripts/run-e2e-test.sh

```diff
@@ -27,7 +27,6 @@ if [[ $mode == "standalone" ]]; then
 fi
 
 if [[ $mode == "single-node" ]]; then
-  export RUST_MIN_STACK=4194304
```

Comment:
This is good to delete, but I'm curious why it's not needed any more?

Reply:
It's already exist in `common.sh`. 😃